.github/workflows/windows.yml

```diff
@@ -0,0 +1,86 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Windows
+
+on:
+  push:
+    branches: [ "winbuild" ]
+    paths-ignore:
+      - 'src/docs/**'
+      - 'src/main/java/**'
+  pull_request:
+    branches: [ "winbuild" ]
+    paths-ignore:
+      - 'src/docs/**'
+      - 'src/main/java/**'
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: Default
+            triplet: x64-windows
+            arch: x64
+            build-type: Debug
+            generator: "Ninja"
+
+    runs-on: windows-latest
+    timeout-minutes: 30
+    name: ${{ matrix.name }}
+    env: 
+      VCPKG_BINARY_SOURCES: "clear;x-gha,readwrite"
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: winbuild
+      - name: Test JAVA_HOME
+        shell: cmd
+        run: |
+            echo rem SPDX-License-Identifier: Apache-2.0> set_java_home.bat
```

Comment:
Is this required in a temporary file?

Reply:
yes because the script uses maven build the test.jar file.

Reply:
Why is the license header required here?

Reply:
Otherwise we get an error like:
[ERROR] Failed to execute goal org.apache.rat:apache-rat-plugin:0.16.1:check (rat-check) on project commons-daemon: Too many files with unapproved license: 1 See RAT report in: /home/jfclere/TMP/TMP/commons-daemon/target/rat.txt

Reply:
Got it. Tx.